common/internal-types/src/tickets.rs

```diff
@@ -19,6 +20,11 @@ const ENCODED_TICKET_LENGTH: usize = 64;
 /// convertible to IEEE754 double-precision and vice versa
 const ENCODED_WIN_PROB_LENGTH: usize = 7;
 
+/// Define the selector for the redeemTicketCall to avoid importing
+/// the entire hopr-bindings crate for one single constant.
+/// This value should be updated with the function interface changes.
+const REDEEM_CALL_SELECTOR: [u8; 4] = [252, 183, 121, 111];
```

Comment:
Can we have a unit test in the `bindings` crate (or anywhere else where we actually do need to import bindings), to verify that this constant is consistent?

Also for this, we'll need it to make it `pub`.

Reply:
forgot to mention: The unit test was added in the internal-types crate

Reply:
Added in `chain/types/src/constants.rs`